client/a8c-for-agencies/components/items-dashboard/items-dataviews/interfaces.ts

```diff
@@ -3,7 +3,7 @@ import { ReactNode } from 'react';
 export interface ItemsDataViewsType< T > {
 	items: T[] | undefined;
 	pagination: DataViewsPaginationInfo;
-	searchLabel?: string;
```

Comment:
The `searchLabel` field is used somewhere

Reply:
Can you please check now? I have rebased it with the latest trunk.